network/endpoint_windows.go

```diff
@@ -43,7 +43,7 @@ const (
 	apipaHostLevelEndpoint = "apipa_hostlevel_endpoint"
 
 	// apipa host level ip
-	apipaHostLevelIP = "169.254.169.2"
+	apipaHostLevelIP = "169.254.255.194"
```

Comment:
why this change is required?
 
@ashvindeodhar will this affect windows multitenancy hostnc connectivity?

Reply:
HLIP prefix 169.254.255.192/26 is allocated for dedicated honeycomb use.
The previous apipaHostLevelIP was in different subnet. 

Just FYI, this change is just for honeycomb-dev branch will be never merged to master.

Reply:
ok then its fine.. lgtm.. lets wait for @aegal approval too

Reply:
is the plan to maintain different fork for honeycomb? @vinayakawanti / @aegal

Reply:
nope, this branch won't be used after phase-0 of honeycomb project.